paperqa/agents/__init__.py

```diff
@@ -0,0 +1,500 @@
+from __future__ import annotations
+
+import ast
+import logging
+import operator
+import os
+import shutil
+from datetime import datetime
+from pathlib import Path
+from typing import Any
+
+import yaml
+from typing_extensions import Annotated
+
+from .. import __version__
+from ..utils import get_loop, pqa_directory
+
+try:
+    import anyio
+    import typer
+    from rich.console import Console
+    from rich.logging import RichHandler
+
+    from .main import aagent_query, asearch, configure_agent_logging
+    from .models import AnswerResponse, MismatchedModelsError, QueryRequest
+    from .search import SearchIndex, get_directory_index
+
+except ImportError as e:
+    raise ImportError(
+        '"agents" module is not installed please install it using "pip install paper-qa[agents]"'
+    ) from e
+
+app = typer.Typer()
+
+logging.basicConfig(
+    level="INFO",
+    format="%(message)s",
+    datefmt="[%X]",
+    handlers=[
+        RichHandler(
+            rich_tracebacks=True,
+            markup=True,
+            show_path=False,
+            show_level=False,
+            console=Console(force_terminal=True),
+        )
+    ],
+)
+
+logger = logging.getLogger(__name__)
+
+
+def get_file_times(path: os.PathLike | str) -> dict[str, str]:
```

Comment:
```suggestion
def get_file_timestamps(path: os.PathLike | str) -> dict[str, str]:
```

Let's be more explicit on what "times" means